spec/spec_helper.rb

```diff
@@ -1,7 +1,7 @@
 $LOAD_PATH.unshift File.join(__dir__, '../lib')
 $LOAD_PATH.unshift File.join(__dir__, 'support')
 
-require 'acme-client'
+require 'acme/client'
```

Comment:
Yes, for compatibility reasons with for example bundler's auto require. But the recommended require line is like this, see http://guides.rubygems.org/name-your-gem/